internal/settings/settings.go

```diff
@@ -9,7 +9,8 @@ import (
 )
 
 type ExperimentalFeatures struct {
-	ValidateOnSave bool `mapstructure:"validateOnSave"`
+	ValidateOnSave            bool `mapstructure:"validateOnSave"`
+	PrefillRequiredAttributes bool `mapstructure:"prefillRequiredAttributes"`
```

Comment:
In line with my comments to your other PR, I think that the name here should reflect that we're completing both attributes and blocks.